src/en/charms-deploying.md

```diff
@@ -237,7 +259,7 @@ juju retry-provisioning 3 27 5
 
 ## Considerations
 
-Charms are running without any separation, so its entirely possible for Charms
+Charms are running without any separation, so its entirely possible for charms
 to stomp all over each others configuration files and try to bind to the same
```

Comment:
I would just leave this for now, we can fix it later.
As it stands, you can deploy things to containers if you want, even when creating a machine, so it is user's choice to colocate rather than containerise.